apollo-test/src/test/java/com/spotify/apollo/test/ServiceHelperExtensionTest.java

```diff
@@ -19,35 +19,27 @@
  */
 package com.spotify.apollo.test;
 
-import static org.junit.jupiter.engine.descriptor.JupiterEngineDescriptor.ENGINE_ID;
-import static org.junit.platform.engine.discovery.DiscoverySelectors.selectClass;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
```

Comment:
Maybe use `org.hamcrest.MatcherAssert#assertThat()` instead?
`org.junit.Assert#assertThat()` is from JUnit 4 and this is a JUnit 5 test

Reply:
Good point! I'll merge anyway. PR very welcome!